packages/console-types/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+- refactor: extend which endpoints routes are displayed when creating endpoints for CRUD Service Views with lookup feature enabled
```

Comment:
please use changeset instead of manually updating the changelog

Reply:
Ok, I removed the changes from `CHANGELOG.md`, so that they be added via `pnpm changeset` when the tag will be created

Reply:
I dont see the changeset file in the `.changeset` directory thought 🤔

Reply:
Sorry, I understood it had to be created after the PR was merged. It should be now available